### PR TITLE
Format source code with ESLint + Prettier

### PR DESCRIPTION
**Pull request by NicoPennec**
**Problem**
The source code is not harmonized and may contain auto-detectable errors

**Solution**
Set ESLint to fix code issues and Prettier to format the source code

### CODE REVIEW

File: src/components/cells/DescriptionCell.vue [hunk 1]
```diff
@@ -79,25 +72,23 @@ export default {
   },
 
   computed: {
-    ...mapGetters([
-    ]),
+    ...mapGetters([]),
 
-    content () {
+    content() {
```
Comment (frankrousseau) [inline src/components/cells/DescriptionCell.vue:77]: This rule is not good. It's not coherent with the rest and I'm not sure it works.

File: src/components/cells/DescriptionCell.vue [hunk 2]
```diff
@@ -153,7 +144,7 @@ td {
 
 .description-cell .tooltip {
   background-color: $white;
-  border-radius: .5rem;
+  border-radius: 0.5rem;
```
Comment (frankrousseau) [inline src/components/cells/DescriptionCell.vue:147]: Note sure about this one either.

File: src/components/cells/DescriptionCell.vue (hunk 1 above)
Comment (NicoPennec) [reply src/components/cells/DescriptionCell.vue:77]: This rule is forced by Prettier 2.x, no options are available to disable this one, but the JS code is still valid.

File: src/components/cells/DescriptionCell.vue (hunk 2 above)
Comment (NicoPennec) [reply src/components/cells/DescriptionCell.vue:147]: This rule is forced by Prettier too. The zero will be removed by the build process anyway.

**Comment by NicoPennec** [comment]
Too much git conflicts... I created a new cleaner PR here #1038

**Pull request closed without merging**